homeassistant/components/template/light.py

```diff
@@ -335,3 +391,81 @@ def template_light_startup(event):
                         self._name,
                         ex,
                     )
+
+    async def update_temperature(self):
+        """Update the temperature from the template."""
+        if self._temperature_template is not None:
+            try:
+                temperature = int(self._temperature_template.async_render())
+            except TemplateError as ex:
+                _LOGGER.error(ex)
```

Comment:
You need to mention that it's for the temperature template or no one will know where the error happened.

Reply:
Done.